### PR TITLE
Replace MvcSample with MvcSandbox

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
As per #3529.

### CODE REVIEW

**Comment by rynowak** [comment]
There was one thing we remembered this week that has non-zero value in the MVC sample. Doug knows. Ask Doug. #YouJustGotDouged

File: samples/MvcSandbox/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+
+            return services.BuildServiceProvider();
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
```
Comment (Eilon) [inline samples/MvcSandbox/Startup.cs]: Maybe register the Console logger here? Probably useful for the sandbox.

**Comment by Eilon** [comment]
Otherwise, :shipit:

File: samples/MvcSandbox/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "ASP.NET",
+  "version": "0.0.0",
+  "devDependencies": {
+    "gulp": "3.8.11",
+    "gulp-concat": "2.5.2",
+    "gulp-cssmin": "0.1.7",
+    "gulp-uglify": "1.2.0",
+    "rimraf": "2.2.8"
```
Comment (dougbu) [inline samples/MvcSandbox/package.json]: Why does this project need `rimraf`?

**Comment by dougbu** [comment]
Also update the `Mvc.sln` file. Then :shipit:

File: test/Microsoft.AspNet.Mvc.FunctionalTests/MvcSampleTests.cs [hunk 3]
```diff
@@ -12,9 +12,9 @@
 
 namespace Microsoft.AspNet.Mvc.FunctionalTests
 {
-    public class MvcSampleTests : IClassFixture<MvcTestFixture<MvcSample.Web.Startup>>
+    public class MvcSandboxTests : IClassFixture<MvcTestFixture<MvcSandbox.Startup>>
```
Comment (dougbu) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/MvcSampleTests.cs]: Should rename this file as well.

**Pull request merged**